### PR TITLE
Update tutorial

### PR DESCRIPTION
**Pull request by nvasilakis**

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -120,58 +156,46 @@ time $PASH_TOP/pa.sh $PASH_TOP/evaluation/intro/hello-world.sh
 
 #### A More Interesting Script: Demo Spell
 
-We will use `demo-spell.sh` --- a pipeline based [on the original Unix spell program](https://dl.acm.org/doi/10.1145/3532.315102) by Johnson --- to confirm that the infrastructure works as expected.
-
-First, we need to setup the appropriate input files for this script to execute:
-
+First, `cd` to `$PASH_TOP/evaluation/intro`:
```
Comment (angelhof) [inline docs/tutorial.md]: Wait, isn't demo-spell very similar log analysis that was shown above?

File: docs/tutorial.md [hunk 2]
```diff
@@ -482,6 +492,9 @@ PaSh’s library currently several aggregators, many of which are usable by more
 
 ## Experimental Evaluation
 
+_Most benchmark sets in the evaluation infrastructure include a `input/setup.sh` script for fetching inputs and setting up the experiment appropriately._
+See [Running other script]() later.
+
 This section provides detailed instructions on how to replicate parts of the experimental evaluation of the system.
 
 Note that input files that are used as inputs for this script are generated using the `gen*` scripts in [evaluation/scripts/input/](../evaluation/scripts/input/).
```
Comment (angelhof) [inline docs/tutorial.md:504]: This is not correct anymore right?

Comment (nvasilakis) [reply docs/tutorial.md:397]: Yeah! I haven't started a pass over the evaluation yet though, this is just what was there already.

File: docs/tutorial.md (hunk 1 above)
Comment (nvasilakis) [reply docs/tutorial.md]: That part is just an intro, it doesn't include any running, but we can discuss how we want to structure it.

File: docs/tutorial.md (hunk 2 above)
Comment (angelhof) [reply docs/tutorial.md:397]: oh cool nice

**Review by angelhof — APPROVED** [review]

**Pull request merged**